docs/core-tech/transaction.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 6
+title: Transaction
+sidebar_label: Transaction
+---
+
+A transaction refers to the act of transferring an entity from one address to another within the network. It is a fundamental operation that enables the exchange of Alephium's native cryptocurrency ALPH, or any other tokens or contract data.
```

Comment:
Other than the value transfer aspect of the transaction, we should probably capture the execution of the smart contract as well.

Reply:
what about: 

> A transaction in Alephium denotes the process of transferring an entity, such as Alephium's native cryptocurrency ALPH, tokens, or contract data, from some addresses to other ones within the network. Beyond facilitating the exchange of value, transactions also play a pivotal role in executing smart contracts. These contracts allow for the implementation of programmable logic, enabling a diverse range of functionalities and decentralized applications on the Alephium blockchain.

Totally no written with ChatGPT :speak_no_evil: :see_no_evil: :hear_no_evil:

Reply:
Maybe replace entity by asset ? Data can be considered an asset no ? It's just the word entity feels weird to me in this context.